devops/scripts/benchmarks/main.py

```diff
@@ -151,6 +152,33 @@ def collect_metadata(suites):
     return metadata
 
 
+def update_metadata_for_cpu_count(metadata, runs) -> dict[str, BenchmarkMetadata]:
```

Comment:
I really dislike this hack :) I would be OK with deleting the old CPU count data and implementing it differently in a way that works better.

Reply:
I've removed the hack and created two metadata objects upfront for each `SubmitKernel` benchmark with respective display names